Dockerfile

```diff
@@ -1,19 +1,9 @@
-FROM php:7.3.15-apache-buster
+FROM php:8.1-apache-bullseye
 
 LABEL maintainer="samanthaj@orangehrm.com"
```

Comment:
@samanthajayasinghe 
@kavithagurusamy 

is it okay to use `opensource@orangehrm.com `